backend/app/api/v1/endpoints/airflow.py

```diff
@@ -29,8 +29,8 @@ async def airflow(pretty: bool = False):
 
 
 
-@router.post('/api/active')
-@router.get('/api/active')
+@router.post('/api/v1/vactive')
+@router.get('/api/v1/vactive')
```

Comment:
Can we send a warning message to this API call?
_This API will be deprecated in the v2 version._